src/components/Navbar.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Nav = () => (
+  <header>
+    <h1>Bookstore CMC</h1>
+    <nav>
+      <Link to="/">Books</Link>
+      <Link to="/categories">Calculator</Link>
```

Comment:
- [ ] May you mind "Categories"? Please 🙏🏽 correct the "Calculator" link for a better user interface.